recipes/quantumespresso/v7.3.1/gh200/repo/packages/nvpl-blas/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
Should we move the `nvpl-blas` and `nvpl-lapack` to the site repo, for all uenv on GH200 to use?